src/__tests__/ReactHardwareComponent-test.js

```diff
@@ -25,55 +30,45 @@ describe('ReactHardwareComponent', () => {
     });
   });
 
-  describe('receiveComponent', function() {
-    let gen;
-    beforeEach(function() {
-      const NodeStub = function(initialProps) {
-        this._currentElement = {props: initialProps};
-        this._rootNodeID = 'test';
-      };
-      NodeStub.prototype = new ReactHardwareComponent({
-        validAttributes: {
-          pin: true,
-          value: true,
-        },
+  describe('pin', () => {
```

Comment:
do we plan on attending tests for more than just the `pin` right now? If not I’m okay leaving this in the top-level `describe` and minimizing the diff.